code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -17,8 +17,10 @@
 
 	health = species.total_health - oxy_l - tox_l - clone_l - total_burn - total_brute
 
+#if HUSKING_ON
```

Comment:
`if(config.example_fire_husking_config_name_on)`

Reply:
This has been done.